src/CSS/viewArticles.css

```diff
@@ -0,0 +1,27 @@
+.articles-list {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    gap: 4px;
+}
+
+.article-item {
+    gap: 10px;
+    display: flex;
+    width: 600px;
+    align-items: left;
+    border: 0.5px solid black;
+    border-radius: 5px;
+    padding: 5px;
+    list-style-type: none;
+
+}
+
+.article-item img {
+    height: 150px;
+    width: 150px;
+}
+
+.article-item h3 {
+    flex-grow: 1;
+}
```

Comment:
Please consider responsive design.

Reply:
I'm not sure what responsive design could be done for this expect maybe hiding text overflow?